cmake/ConfigureChecks.cmake

```diff
@@ -676,6 +676,11 @@ endif()
 set(CFG_HEADERS ${CFG_HEADERS} time.h stdio.h math.h)
 
 check_symbol_exists(alarm        "${CFG_HEADERS}" HAVE_ALARM)
+if(HAVE_ALLOCA_H)
```

Comment:
The `if` statement is not needed. `stdlib.h` is already added to `CFG_HEADERS`
See https://github.com/tony/python-cmake-buildsystem/blob/a4c25ff1b02063f4994e998ec72a2d10a541a259/cmake/ConfigureChecks.cmake#L655